src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -376,22 +376,12 @@ export class SearchInterface extends RootComponent implements IComponentBindings
     enableCollaborativeRating: ComponentOptions.buildBooleanOption({ defaultValue: false }),
 
     /**
-     * Specifies whether to filter duplicates in the search results.
+     * Whether to filter out duplicates, so that items resembling one another only appear once in the query results.
      *
-     * Setting this option to `true` forces duplicates to not appear in search results. However, {@link Facet} counts
-     * still include the duplicates, which can be confusing for the end user. This is a limitation of the index.
-     *
-     * **Example:**
-     *
-     * > The end user narrows a query down to a single item that has a duplicate. If the enableDuplicateFiltering
-     * > option is `true`, then only one item appears in the search results while the Facet count is still 2.
-     *
-     * **Note:**
-     *
-     * > It also is possible to set this option separately for each {@link Tab} component
-     * > (see {@link Tab.options.enableDuplicateFiltering}).
```

Comment:
There's similar documentation on the `Tab` component, that we should also adapt. 

And since it's possible to set the option in "two" different place, it seems like we should not remove that information ?

Reply:
Whoops, you're right! I cut a bit too much here 😅